.docker/autoline/docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '3'
+services:
+  autoline_mariadb:
+    image: mariadb
+    container_name: 'autoline_mariadb'
+    build:
+      context: ./
+      dockerfile: database/Dockerfile
+    hostname: mariadb
+    ports:
+      - "3307:3306"
+    volumes:
+      - autoline_mariadb_data:/var/lib/mysql
+    environment:
+      - "MYSQL_DATABASE=autoline"
+      - "MYSQL_USER=autoline"
+      - "MYSQL_PASSWORD=secret"
+      - "MYSQL_ROOT_PASSWORD=secret"
```

Comment:
It's better to include the backend env file than to use the inline variables. In that way, there will be single source of truth for the DB connection and there will be no possibility to use wrong credentials in the backend